package.json

```diff
@@ -2,7 +2,7 @@
   "name": "fabric8-analytics",
   "displayName": "Dependency Analytics",
   "description": "Insights about your application dependencies: Security, License compatibility and AI based guidance to choose appropriate dependencies for your application.",
-  "version": "0.0.13",
+  "version": "0.0.14",
```

Comment:
Do it in a dedicated PR., not as part of this.

Reply:
fixed it